resources/views/home.blade.php

```diff
@@ -0,0 +1,20 @@
+<x-app-layout>
+    @guest
+        @include('content.welcome')
+    @endguest
+    <x-news.carousel/>
+    <?php
+    renderFinishedClaimsComponent(5);
+    RenderActivePlayersComponent();
+    ?>
+    <x-user.online-count-chart/>
+    <?php
+    renderNewClaimsComponent(5);
+    RenderRecentForumPostsComponent();
+    ?>
+    @slot('sidebar')
+        @include('content.top-links')
+        <x-event.aotw/>
+        @include('content.static-data')
+    @endslot
+</x-app-layout>
```

Comment:
Headers seem to have lost their underlines. It's probably okay, just tripping something in my OCD.
![image](https://user-images.githubusercontent.com/32680403/215283647-b43e9746-49c9-42ca-aee7-d7dc80809a3a.png)

Reply:
Whoops - I planned on removing the underlines at _some_ point, not in this PR though.
The overall layout has to match such a change. Reverted for consistency.